packages/ember-glimmer/tests/utils/abstract-test-case.js

```diff
@@ -78,6 +86,7 @@ export class RenderingTest extends TestCase {
     let owner = this.owner = buildOwner();
     let env = this.env = new Environment({ dom, owner });
     this.renderer = new Renderer(dom, { destinedForDOM: true, env });
+    this.$ = jQuery;
```

Comment:
ah, this is not what I meant, I mean something like how `this.$()` works in components, but I can fix it